frontend/pages/menu.tsx

```diff
@@ -2,25 +2,61 @@
 import { GetStaticProps } from "next";
 import { sanityClient } from "../lib/sanityClient";
 import { MenuItem } from "../types"; // Define your types if needed
+import { Card, CardBody, Heading, Stack, Text, Button, Image, CardFooter } from '@chakra-ui/react';
 
 interface MenuProps {
   menuItems: MenuItem[];
 }
 
 const MenuPage = ({ menuItems }: MenuProps) => {
   return (
-    <div>
-      <h1>Our Menu</h1>
-      <ul>
+    <div className="bg-black min-h-screen py-12 pt-40">
+      <h1 className="text-center text-white text-4xl font-bold mb-8">Choose Our Menu</h1>
+      <div className="container mx-auto grid grid-cols-[repeat(auto-fit,minmax(300px,1fr))] gap-5 mt-5 justify-items-center px-5">
```

Comment:
in h1 tag you set mb-8, and in div you set mt-5. 
It will be better set only margin once in one place.